lib/beaker/host/unix.rb

```diff
@@ -64,17 +64,10 @@ def self.aio_defaults
       h = Beaker::Options::OptionsHash.new
       h.merge({
         'user'                  => 'root',
-        'group'                 => 'puppet',
         'puppetserver-confdir'  => '/etc/puppetlabs/puppetserver/conf.d',
         'puppetservice'         => 'puppetserver',
-        'puppetpath'            => '/etc/puppetlabs/puppet',
-        'puppetconfdir'         => '/etc/puppetlabs/puppet',
-        'puppetvardir'          => '/opt/puppetlabs/puppet/cache',
-        'puppetbin'             => '/opt/puppetlabs/puppet/bin/puppet',
         'puppetbindir'          => '/opt/puppetlabs/bin',
-        'systembindir'          => '/usr/bin',
-        'hieradatadir'          => '/etc/puppetlabs/code/hieradata',
-        'hieraconf'             => '/etc/puppetlabs/code/hiera.yaml',
+        'privatebindir'         => '/usr/bin',
```

Comment:
I may have missed context, but I expected aio's `privatebindir` to be `/opt/puppetlabs/puppet/bin`.